firmware/misterynano_fw/menu.c

```diff
@@ -176,15 +176,13 @@ static const char system_form_vic20[] =
   "System,0|2;"                         // return to form 0, entry 2
   // --------
   "L,Joyport 1:,Retro D9|USB #1|USB #2|NumPad|DualShock|Mouse|Paddle|Off,Q;" // Joystick port 1 mapping
-  "L,Joyport 2:,Retro D9|USB #1|USB #2|NumPad|DualShock|Mouse|Paddle|Off,J;" 
-  "L,REU 1750:,Off|On,V;"                
   "L,c1541 ROM:,Dolphin DOS|CBM DOS|Speed DOS P|Jiffy DOS,D;"  // c1541 compatibility
-  "L,Audio filter:,Off|On,U;"
-  "L,Turbo mode:,Off|C128|Smart,X;"
-	"L,Turbo speed:,2x|3x|4x,Y;"
+  "L,ExtRAM $0400(3KB):,Off|On,U;"
```

Comment:
I wonder if it would be more intuitive to have as single memory menu entry with entries like "3k", "8k", 16k" and so on. I understand that this would not include weird setups with gaps. But are these useful, anyway?

Reply:
i am not sure if i stay with MISTer components (there is DDR !) or still swap to MIST (pure DRAM). In MISTer they have the RAM expansion like that defined for whatever reason.

Reply:
DDR? MiSTer has the same SDR-SDRAM on its expander board that the MiST uses. It's just a little bigger. But what has the RAM type to do with the way this is presented in the menu`?

Reply:
RAM Expansion presentation i could change if too confusing.

My point was if effort in replacing DE10-nano DDR3 memory interface
https://github.com/MiSTer-devel/VIC20_MiSTer/blob/master/rtl/ddram.sv

to TN20k environment then other core components like MIST VIC20 could be more useful and easier to handle